notebook/gen_ai_jss.ipynb

```diff
@@ -0,0 +1,818 @@
+    "id": "960505627ddf"
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "metadata": {
+    "id": "PyQmSRbKA8r-"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import polling2\n",
+    "import re\n",
+    "import time\n",
+    "\n",
+    "from alive_progress import alive_bar\n",
+    "from google.cloud import aiplatform\n",
+    "from google.cloud import bigquery\n",
+    "from google.cloud import logging\n",
+    "from google.cloud import storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download test data\n",
+    "\n",
+    "This Jump Start Solution uses data from [arXiv.org](https://arxiv.org/) to demonstrate the summarization capabilities of Vertex AI. arXiv, through [Kaggle.com](https://www.kaggle.com/datasets/Cornell-University/arxiv) has made many scholarly papers available, free of charge, from a Google Cloud Storage bucket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404001v1.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404002v1.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404003v2.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404004v1.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404005v1.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404007v1.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404008v1.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404009v3.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404010v2.pdf\n",
+      "gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404011v2.pdf\n"
+     ]
+    }
+   ],
+   "source": [
+    "# List all the comparative linguistics papers from Cloud Storage\n",
+    "! gsutil ls gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**TODO: Remove before final merge.**\n",
+    "\n",
+    "Known good PDFs:\n",
+    "\n",
+    "+ 9404001v1.pdf\n",
+    "+ 9404002v1.pdf\n",
+    "+ 9404003v2.pdf\n",
+    "+ 9404008v1.pdf\n",
+    "+ 9404011v2.pdf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 103,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Copying gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/9404002v1.pdf...\n",
+      "/ [1 files][196.1 KiB/196.1 KiB]                                                \n",
+      "Operation completed over 1 objects/196.1 KiB.                                    \n"
+     ]
+    }
+   ],
+   "source": [
+    "filename = '9404002v1'\n",
+    "file_uri = f'gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/{filename}.pdf'\n",
+    "\n",
+    "# Create a local folder and download some test PDFs\n",
```

Comment:
Can we sneak peak the files here? Can insert an image here.

Reply:
We want a before & after experience. (PDF v.s. output text)

Reply:
We can probably show part of one image as a "before." These PDF files are usually > 7 pages long.

Reply:
Let's add an image as a v2 enhancement.